Gemfile.lock

```diff
@@ -332,9 +328,6 @@ GEM
     tzinfo (1.2.9)
       thread_safe (~> 0.1)
     unicode-display_width (1.7.0)
-    unicorn (5.8.0)
```

Comment:
what was unicorn doing before, and is it being removed because it was a dependency on something else or its being removed for some other reason?
Just asking as gunicorn for example is what I know we use on another project only for the production server, so I'm not sure if this was doing something similar in which case it'd be good to double check

Reply:
This is the Gemfile.lock file, not the Gemfile file, so unicorn wasn't being used. I think it's actually my mistake from the PDF rendering spike that it's in here at all.